scripts/rollup/results.json

```diff
@@ -25,28 +25,28 @@
       "gzip": 6703
     },
     "react-dom.development.js (UMD_DEV)": {
-      "size": 613875,
-      "gzip": 141842
+      "size": 614249,
```

Comment:
Should we update this file or leave it until the next release we build?

Reply:
Doesn't really matter. :-)